packages/core/entity/FileDetail/RenderZarrThumbnailURL.ts

```diff
@@ -119,11 +127,17 @@ export async function renderZarrThumbnailURL(zarrUrl: string): Promise<string |
                 const u16data = lowestResolutionView.data as Uint16Array;
 
                 // Normalize Data to improve image visibility.
-                const min = Math.min(...u16data);
-                const max = Math.max(...u16data);
+                let min = Infinity;
```

Comment:
This could be written this way to be more javascripty: 
```                
const { min, max } = u16data.reduce(
    (acc, val) => ({
        min: Math.min(acc.min, val),
        max: Math.max(acc.max, val),
    }),
    { min: Infinity, max: -Infinity }
);
```

Reply:
As an old C++ programmer, I can't resist.  You can do performance tests with these functional constructs that show they are much much slower than a simple for loop with integer index. For times when performance matters, like iterating over long typedarrays.

Reply:
Yes, I didn't test it here but that was my understanding. There is some overhead of creating and calling a function for every pixel. Would be interesting to check...